### DISCUSSION TITLE
Google auth

### DISCUSSION THREAD

**Discussion by Pheggas** [immich-app/immich#2658] [Q&A] [answered]
Hello. I would like to spin up Immich with google auth but i don't see the option to login with google auth in mobile app of Immich. Is there such possibility or it's not implemented yet - and then how could i log in if the google auth isn't possibility in mobile app?

**Comment by bo0tzz** [comment]
You can use oauth via Google: https://immich.app/docs/administration/oauth

**Reply by Pheggas** [reply]
And how the Immich app behaves when i set up google oauth via Keycloak?

When i enter the server URL, will it redirect me to the web where i authenticate with my google account and it will log me in to the Immich app?

**Reply by jrasm91** [reply, marked as answer]
Yes
